test/complex.jl

```diff
@@ -208,8 +208,7 @@ end
 end
 
 @testset "log" begin
-    #  log(conj(z)) = conj(log(z))
-
+    # log(conj(z)) = conj(log(z))
```

Comment:
Perhaps incorporate the comment in the testset description and nix the comment altogether?